todoapp-main/src/main/java/ru/todoapp/configuration/KafkaConfig.java

```diff
@@ -10,9 +10,7 @@
 import org.springframework.kafka.core.KafkaAdmin;
 import org.springframework.kafka.core.KafkaTemplate;
 import org.springframework.kafka.core.ProducerFactory;
-import ru.todoapp.model.dto.PingRequestDTO;
-import ru.todoapp.model.dto.RequestResultDTO;
-import ru.todoapp.model.dto.RegisterRequestDTO;
+import ru.todoapp.model.dto.*;
```

Comment:
Don't use wildcard imports (with '*'). You cans use them only on tests.
Have you applied a codestyle file from this repo to your's idea?

Reply:
idea_set_code_style.xml?
I think we did... then we were merging this changes to that branch